static/_redirects

```diff
@@ -1,8 +1,270 @@
+/integrations/postresql/postresql-metrics/ https://docs.virtana.com/en/postresql-metrics.html 301!
+/integrations/powerdns/ https://docs.virtana.com/en/powerdns.html 301!
+/integrations/process-resources-collector/ https://docs.virtana.com/en/process-resources.html 301!
+/integrations/puppet/ https://docs.virtana.com/en/puppet.html 301!
+/integrations/puppetdb/ https://docs.virtana.com/en/puppetdb.html 301!
+/integrations/python/ https://docs.virtana.com/en/python.html 301!
+/integrations/rabbitmq/ https://docs.virtana.com/en/rabbitmq.html 301!
+/integrations/rabbitmq/rabbitmq-metrics/ https://docs.virtana.com/en/rabbitmq-metrics.html 301!
+/integrations/redis/ https://docs.virtana.com/en/redis.html 301!
+/integrations/redis/redis-metrics/ https://docs.virtana.com/en/redis-metrics.html 301!
+/integrations/riemann/ https://docs.virtana.com/en/riemann.html 301!
+/integrations/snmp-interface-collector/ https://docs.virtana.com/en/snmp.html 301!
+/integrations/snmp-collector/ https://docs.virtana.com/en/snmp-interface.html 301!
+/integrations/sso/ https://docs.virtana.com/en/sso.html 301! 
+/integrations/sso/sso-azure/ https://docs.virtana.com/en/azure-sso.html 301! 
+/integrations/sso/sso-jumpcloud/ https://docs.virtana.com/en/jumpcloud-sso.html 301!
+/integrations/sso/sso-okta/ https://docs.virtana.com/en/okta-sso.html 301!
+/integrations/solr/ https://docs.virtana.com/en/solr.html 301!
+/integrations/tcp-collector/ https://docs.virtana.com/en/tcp.html 301!
+/integrations/tcp-collector/tcp-metric-names/ https://docs.virtana.com/en/allowed-metric-names.html 301!
+/integrations/tomcat/ https://docs.virtana.com/en/tomcat.html 301!
+/integrations/user-scripts-collector/ https://docs.virtana.com/en/user-scripts.html 301!
+/api/ https://docs.virtana.com/en/api.html 301!
+/api/api-alerts/ https://docs.virtana.com/en/alerts-api.html 301!
+/api/api-cli/ https://docs.virtana.com/en/cli.html 301!
+/api/api-dashboards/ https://docs.virtana.com/en/dashboards-api.html 301!
+/api/api-datasources/ https://docs.virtana.com/en/datasources-api.html 301!
+/api/api-ingest/ https://docs.virtana.com/en/ingest-api.html 301!
+/api/api-metrics/ https://docs.virtana.com/en/metrics-api.html 301!
+/api/api-notifications/ https://docs.virtana.com/en/notifications-api.html 301!
+/api/api-packages/ https://docs.virtana.com/en/packages-api.html 301!
+/api/api-policies/ https://docs.virtana.com/en/policies-api.html 301!
+/api/api-users/ https://docs.virtana.com/en/users-api.html 301!
+/api/api-widgets/ https://docs.virtana.com/en/widgets-api.html 301!
+/api/api-elements/ https://docs.virtana.com/en/elements-api.html 301!
 /right-sizing/ec2-recommendations   /right-sizing/reports-ec2-recommendations/
 /right-sizing/resource-utilization   /right-sizing/reports-resource-utilization/
 /right-sizing/reports-ec2-reservations   /purchase-planning/reports-ec2-reservations/
 /right-sizing/resource-utilization   /right-sizing/reports-ec2-recommendations/
```

Comment:
This should redirect to the new link that you have above:   https://docs.virtana.com/en/ec2-recommendations.html

```suggestion
/right-sizing/resource-utilization   https://docs.virtana.com/en/ec2-recommendations.html 301!
```